modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -45,16 +45,18 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: image-registry-storage <1>
+  namespace: openshift-image-registry <2>
 spec:
   accessModes:
-  - ReadWriteOnce <2>
+  - ReadWriteOnce <3>
   resources:
     requests:
-      storage: 100Gi <3>
+      storage: 100Gi <4>
 ----
 <1> A unique name that represents the `PersistentVolumeClaim` object.
-<2> The access mode of the PersistentVolumeClaim. With `ReadWriteOnce`, the volume can be mounted with read and write permissions by a single node.
-<3> The size of the PersistentVolumeClaim.
+<2> The namespace for the `PersistentVolumeClaim` object, which is `openshift-image-registry`.
```

Comment:
Is the project for a PVC always `openshift-image-registry`, or just in this example?

Reply:
Yes, it should always be configured for openshift-image-registry (as per the bug description).